ci/audit.yml

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
```

Comment:
This is the content that should be in `ci/scripts/audit.sh`

Reply:
Ok - looks like the automated project build isn't generating these correctly.